spec/configuration_spec.rb

```diff
@@ -1,4 +1,6 @@
 # encoding: utf-8
+require 'json'
```

Comment:
I'm not sure why this was added as I don't see any explicit uses of the json gem/library in this file.

Reply:
I'm not sure why this comment was marked as resolved. This file should require `spec_helper` like before because spec_helper.rb contains global setup/configuration for tests. Additionally, it isn't clear why to me why you're requiring json in this file.

Reply:
Apologies - I applied the change the wrong way round for some reason! Now removed `json` and restored `spec_helper`.